docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,171 @@
+# Release 465 (20 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for parameter of `bigint` type for the {func}`repeat` function. ({issue}`22867`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Expose query start count and rate metrics for resource groups. ({issue}`23984`)
+* {{breaking}} Update the required [](jvm-config). ({issue}`24161`)
```

Comment:
This is too vague. What's required?

Also, what change in this release requires users to specify new JVM config options? It doesn't seem like there's anything in this release that introduces a breaking change.

Reply:
The security manager stuff was added .. I also updated the old release notes to link to the jvm config when we made 23 a requirement .. so maybe we can drop it here.. but now is when we found out... soo .. not sure

Reply:
Removed - also others.